### ISSUE TITLE
It's not possible to configure the BASE_URL for a domain starting with board

### ISSUE THREAD

**Issue by rafaelchavesfreitas** [nikku/wuffle#154]
__Describe the Bug__

It's not possible to configure the BASE_URL for a domain starting with board, becouse `https://board.myproject.com` contains `/board` and the function `checkBaseUrl` of file run.js do this:

```JS
 function checkBaseUrl() {

    const baseUrl = process.env.BASE_URL;

    if (baseUrl) {
      if (baseUrl.includes('/board')) {
        return error('env.BASE_URL must point to the root of the webapp');
      }
```

I think that you can change the code to first check if the baseUrl format is not valid, then check why, somethink like this:

```JS
 function checkBaseUrl() {

    const baseUrl = process.env.BASE_URL;

    if (baseUrl && !baseUrl.match(/^https?:\/\/(\.?\w+)+$/)) {
      if (baseUrl.includes('/board')) {
        return error('env.BASE_URL must point to the root of the webapp');
      }

```

__Steps to Reproduce__

try to configure the BASE_URL for a domain starting with board

__Environment__

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: [e.g. Windows 7]
- Board version: [e.g. 1.0.0]

**Comment by nikku** [comment]
Good catch. I think the check can be further simplified. It should basically prevent users from setting `{TASK_BOARD}/board` as the `BASE_URL` over `{TASK_BOARD}`.

**Comment by petja-laitila** [comment]
Hi,

We are running version of wuffle on address that starts with board and we've been able to get around this problem having IP as base_url. However latest version is refusing to start and I'm wondering if it's related to having IP in there.  I'm having difficulties to see what is causing the hangup as we're running this in container and not much useful information is coming out.

**Comment by nikku** [comment]
Not sure which release things worked, and which it started to fail (you don't provide that information), hence I cannot answer your question.

What I'd suggest is that we fix the original issue, and then you'll be fine.

**Comment by petja-laitila** [comment]
I'm actually not exactly sure of version since it's been a while since last update (it is latest from that time), but it is from 18 months in the past. Our problem might not be this issue though. But current latest image won't start with the same configurations we had for that one. I'm sorry I can give such limited information. edit. our problem came from wuffle.config.js file where we still had module.exports = { etc. changing to export default { etc.fixed it.
